### DISCUSSION TITLE
Arabic language support

### DISCUSSION THREAD

**Discussion by abod2data** [frappe/helpdesk#1868] [Q&A]
Arabic language and RTL not supported in frappe Helpedesk 
Any help with that?

**Comment by ashrafayad** [comment]
find ~/frappe-bench/apps/helpdesk -type f -exec sed -i 's/InterVar/system-ui/g' {} +
bench build --app helpdesk

This will change the InterVar to system-ui in all helpdesk files, easy and fast.
